pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -0,0 +1,49 @@
+{stdenv, fetchurl, unzip, libX11, libcxxabi, glib, xorg, qt4, fontconfig, phonon, freetype, zlib, libpng12, libICE, libXrender, cups}:
+
+stdenv.mkDerivation {
+  name = "wpsoffice-10.1.0.5503";
+
+
+  src = fetchurl {
+    name = "wps-office.tar.gz";
```

Comment:
it would be better not to have a fixed name. You can use `name = "${name}.tar.gz";` to make it relative to the parent's name.